manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -84,6 +84,17 @@ spec:
             runAsGroup: 0
             runAsUser: 0
             runAsNonRoot: false
+            capabilities:
+              drop:
+              - ALL
+              add:
+              - NET_ADMIN
+              # TODO this, `privileged`, etc need to be reworked - both sidecar (repair) and ambient require SYS_ADMIN
+              # and istio-cni is *always* privileged to some degree, regardless of the `privileged` flag
+{{- if .Values.cni.ambient.enabled }}
```

Comment:
Why does IF start here at not L86? I don't think we use NET_ADMIN outside of ambient

Reply:
Current `master` chart has NET_ADMIN already unconditionally, so I kept it: https://github.com/istio/istio/blob/master/manifests/charts/ztunnel/templates/daemonset.yaml

Can fix in https://github.com/istio/istio/issues/49004, I am not sure where we use it, only that `master` has it.

Reply:
The link is ztunnel, this is cni DS?